hudi-spark-datasource/hudi-spark/src/main/java/org/apache/hudi/cli/BootstrapExecutorUtils.java

```diff
@@ -182,6 +182,13 @@ public void execute() throws IOException {
       checkpointCommitMetadata.put(CHECKPOINT_KEY, Config.checkpoint);
       bootstrapClient.bootstrap(Option.of(checkpointCommitMetadata));
       syncHive();
```

Comment:
Thanks for your suggestion, address it.
Yeah, If If `syncHive` failed, we could call [hive_sync](https://hudi.apache.org/docs/next/procedures/#hive_sync) to avoid rerun the long time bootstrap.

Reply:
Thanks for your suggestion, address it.